Mathlib/Tactic/Linter/TextBased.lean

```diff
@@ -163,6 +169,10 @@ def parse?_errorContext (line : String) : Option ErrorContext := Id.run do
         | "ERR_SEM" => some (StyleError.semicolon)
         | "ERR_TWS" => some (StyleError.trailingWhitespace)
         | "ERR_WIN" => some (StyleError.windowsLineEnding)
+        | "ERR_UNICODE" => do
```

Comment:
Have to tested that this part works? I would think `errorMessage` also contain the actual error message...
Otherwise, I see what you're doing here (parsing the unicode codepoint from an expression `U+NNNN`) and agree with that. Can you add a comment, though - so future readers can also tell this quickly? Thanks!